chingu-voyage4/Geckos-Team-32
# Database

## Purpose
* Merge Branches
* Save videos

## Approach/Fixes
* Merged `Profile` branch
* Created `component to store saved/liked videos
* Updated profile and dash component
* Refactored a PlayVideo component to DRY up code
* Temporarily set state to change user's avatar when logged in
* Hide dashboard upon initial landing, then show after searching
* Created seed data to dev purposes

## Testing
* Download this PR to your local machine
* Run `npm install`
* Load up the server and check the following:
  * Dashboard does not initially show until searching a video, or going to the profile page
  * Dashboard links should route to respective components
  * Profile link for `saved videos` should route to savedvideo component
  * In profile check that you can change your avatar
  * After clicking a video, try saving it by clicking the little black heart icon
  * Checked your saved videos to see that it persists

## Notes
* The saved videos will only work with the server running
* I haven't gone through all the bugs yet, so there may be SOME errors...
* Still need to configure avatar and other profile stuff on backend
* Current `AppRouter` has seeded data in it. You can comment it out when running the server to test functionality with a user login, otherwise keep it when testing stuff while running the dev-server
* Didn't fully check all the styles in tablet/mobile view yet!
